src/inmanta/moduletool.py

```diff
@@ -193,15 +193,22 @@ def freeze(self, outfile, recursive, operator):
         requires = sorted([k + " " + v for k, v in freeze.items()])
         newconfig["requires"] = requires
 
+        close = False
```

Comment:
Shouldn't we put this in a try-finally block

Reply:
Hard to say, do we want to flush the buffer if the write failed?
Does is matter not flushing it?
Do we have to close it even if we know the process will be terminated?

I will add the block just to be sure.